packages/contentful-extension-scripts/template/javascript-entry/src/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  DisplayText,
+  Paragraph,
+  SectionHeading,
+  TextInput,
+  Textarea,
+  FieldGroup,
+  RadioButtonField,
+  Form,
+} from '@contentful/forma-36-react-components';
+import { init, locations } from 'contentful-ui-extensions-sdk';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss';
+import './index.css';
+
+/**
+ * To use this demo create a Content Type with the following fields:
```

Comment:
You could dump your content type to JSON and attach so it can be created with a single HTTP call.